backend/prisma/schema.prisma

```diff
@@ -0,0 +1,50 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
+// Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model User {
+  address      String        @id @unique
+  name         String?
+  transactions Transaction[]
+  companies        Company[]
+}
```

Comment:
You are working on the user model. 

In your task you juste have to care about team.